aya/src/programs/links.rs

```diff
@@ -28,6 +31,25 @@ pub trait Link: std::fmt::Debug + 'static {
     fn detach(self) -> Result<(), ProgramError>;
 }
 
+bitflags::bitflags! {
+    /// Flags passed to [`SockOps::attach()`].
+    #[derive(Clone, Copy, Debug, Default)]
+    pub struct CgroupAttachFlags: u32 {
+        /// Allows the eBPF program to be overridden by ones in a descendent cgroup.
+        const ALLOW_OVERRIDE = BPF_F_ALLOW_OVERRIDE;
+        /// Allows multiple eBPF programs to be attached.
+        const ALLOW_MULTI = BPF_F_ALLOW_MULTI;
+        /// Replaces an existing attachment.
+        const REPLACE = BPF_F_REPLACE;
```

Comment:
Have you tested this works as expected?

Reply:
Please see the **Testing** comment I've added below.  This covers the two flags that have been kept in this PR.